src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/NomisApiService.kt

```diff
@@ -78,5 +78,7 @@ class NomisApiService(
     }
   }
 
+  private fun String.nomisRoleName(): String = take(30)
```

Comment:
So will truncate then, will the UI restrict to 30?

Reply:
No, the UI doesn't restrict to 30 chars.  We allow 100 (stored in auth) but it is just the call to Nomis that truncates to 30.